paper/paper.md

```diff
@@ -0,0 +1,462 @@
+We recorded facilitators teaching the lesson notebooks as if they were live
+coding, then shared them along with a link to the lesson notebooks for students
+to code along with while watching the videos.
+Each video shows the Jupyter notebook alongside the facilitator themselves
+teaching.
+This format allowed students to learn at their own pace, then ask questions and
+practice when we met together virtually. <!-- TODO: should this actually go later in the curriculum section?-->
+
+Our Jupyter notebooks and links to the lesson videos can be accessed on GitHub:
+https://github.com/GWC-DCMB/curriculum-notebooks
+
+# Curriculum
+
+Our curriculum was designed for high school students with no prior coding
+experience who are interested in learning Python programming for data science.
+However, this course material would be useful for anyone interested in learning
+basic programming for data analysis, regardless of age or experience level.
+
+Our curriculum features short lessons to deliver course material in “bite sized”
+chunks, followed by practices to solidify the learners' understanding.
+
+## Learning Objectives
+
+Our learning objectives are:
+
+1. Understand fundamental concepts and best practices in coding.
+2. Apply data analysis to real world data to answer scientific questions.
+3. Create informative summary statistics and data visualizations in Python.
+
+These skills provide a solid foundation for basic data analysis in Python and
+participation in our events exposes students to the many ways coding and
+data science can be used to make large impacts across many disciplines.
+
+## Course Content
+
+Our curriculum design consists of 27 lessons broken up into 5 modules that cover Jupyter notebook setup,
+Python coding fundamentals, use of essential data science packages including
+Pandas and numpy, basic statistical analyses, and plotting using seaborn and
+matplotlib (Figure 1) [@harris_array_2020; @waskom_seaborn_2021;
+@hunter_matplotlib_2007].
+Each of the following modules consists of a lesson notebook, used for teaching
+the concept via live coding, and a practice notebook containing similar
+exercises for the student to complete on their own following the lesson:
+
+Jupyter Notebook Setup:
+
+1. Jupyter Setup
+
+Coding Fundamentals:
+
+2. HelloWorld and Variables
+3. Variables and Types
+4. Lists Intro
+5. Indexing
+6. 2D Lists Intro
+7. 2D Lists Indexing
+8. Logic
+9. Conditionals
+10. Loops I
+11. Loops II
+12. Functions and Methods
+
+Data Science Essentials:
+
+13. Packages
+14. Pandas-Intro
+15. Pandas-Reading
+16. Pandas Subsetting I
+17. Pandas Subsetting II
+18. Dictionaries
+19. Functions
+20. Numpy Intro
+
+Basic Statistical Analysis:
+
+21. Basic Stats I - Averages
+22. Basic Stats II - Percents
+23. Basic Stats III - Correlations
+24. Basic Stats IV - Significance
+
+Plotting:
+
+25. Plotting I - Line Graphs
+26. Plotting II - Scatterplots
+27. Plotting III - Bar Charts and Histograms
```

Comment:
Again with standardizing naming conventions - none of the previous modules are included in lesson names, maybe we ditch the 'Basic Stats I -' and 'Plotting II -' prefixes in these last two modules. We should also make sure all the lessons in the gitthub repo match. Lastly, I think the module should be named Data Visualization instead of Plotting (I implemented this in the suggested figure above)

Reply:
Done